app/views/admin/memberships/new.html.haml

```diff
@@ -1,13 +1,19 @@
-%table
-  %thead
-    %tr
-      %th Supplier
-      %th Actions
-  %tbody
-    - @suppliers.each do |supplier|
-      %tr
-        %td= supplier.name
-        %td
-          = form_for [:admin, @user, Membership.new] do |form|
-            = form.hidden_field :supplier_id, value: supplier.id
-            = form.submit 'Link user'
+.govuk-grid-row
+  .govuk-grid-column-full
+    %fieldset.govuk-fieldset
+      %legend.govuk-fieldset__legend.govuk-fieldset__legend--xl
+        %h1.govuk-fieldset__heading
+          Link to a supplier
```

Comment:
I wonder if this title should include the name of the user being linked or some explanation text beneath?